wagtail/snippets/templates/wagtailsnippets/snippets/create.html

```diff
@@ -12,7 +12,7 @@
 
 {% block actions %}
     <footer class="footer">
-        <nav class="actions actions--primary footer__container w-grid" aria-label="{% trans 'Actions' %}">
+        <nav class="actions actions--primary footer__container" aria-label="{% trans 'Actions' %}">
```

Comment:
Can confirm that this works OK even when there's only a single action

<img width="356" alt="image" src="https://github.com/wagtail/wagtail/assets/6379424/e9c02874-bd45-4de0-9bc4-130299b1e360">

<img width="356" alt="image" src="https://github.com/wagtail/wagtail/assets/6379424/1ee70ecb-58e1-4246-a715-8e57a11aa7da">